examples/traits/pdb/README.md

```diff
@@ -0,0 +1,20 @@
+# Camel-k PDB Trait
+
+This is an example of a Pod Distruption Budget for an integration pod running on the cluster. 
+You can set a  `minumum available` or a `maximum unavailable` pods, it all depends on your eviction policy.
+
+ **Note:** This trait is best used on a multi-node cluster. Also, you can only use one trait command. Either a `minimum available` or a `maximum unavailable`
```

Comment:
Better to tell the trait is more suitable to show its features on a cluster with more than a single node (ie, minikube).